tests/integration/conftest.py

```diff
@@ -25,6 +26,11 @@ def app_config(app_config):
     # add extra global config if you would like to customize the config
     # for a specific test you can chagne create fixture per-directory
     # using ``conftest.py`` or per-file.
+    app_config[
+        "DEBUG"
+    ] = (
+        False
+    )  # Due to flask error it has to be False otherwise Alembic __init__ will fail.
```

Comment:
the style here is weird :/ is it from black?

Reply:
Unfortunately yes. As I  was formatting this completely different.

Reply:
oh I get it is because you have the comment after ``False`` should be in a different line i.e.

```
# Due to flask error it has to be False otherwise Alembic __init__ will fail.
app_config["DEBUG"] = False
```

Reply:
You are right.